lib/internal/test_runner/test.js

```diff
@@ -817,6 +845,7 @@ class Suite extends Test {
   async run() {
     const hookArgs = this.getRunArgs();
 
+    let stopPromise;
```

Comment:
I find this name somewhat confusing, as this is not actually a promise, but a disposable wrapper for a promise...
Maybe `stopPromiseWrapper` etc? WDYT?